schemas/entity/MRImage.schema.tpl.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
we need to discuss what belongs where. I think some of the properties here may go to the device usage for mri scanners.

We need to distinguish between (at least) structural MRI images and functional MRI images.